content/docs/usage/approver-policy.md

```diff
@@ -3,322 +3,12 @@ title: Policy for cert-manager certificates
-      - "spiffe://example.org/ns/*/sa/*"
-    emailAddresses:
-      values:
-      - "*@example.com"
-      required: true
-    isCA: false
-    usages:
-    - "server auth"
-    - "client auth"
-    subject:
-      organizations:
-        values: ["hello-world"]
-      countries:
-        values: ["*"]
-      organizationalUnits:
-        values: ["*"]
-      localities:
-        values: ["*"]
-      provinces:
-        values: ["*"]
-      streetAddresses:
-        values: ["*"]
-      postalCodes:
-        values: ["*"]
-      serialNumber:
-        value: "*"
-  ...
-```
-
-### Constraints
-
-Constraints is the block that is used to limit what attributes the request can
-have. If a constraint is not defined, then the attribute is considered "allow
-all".
-
-Below is an example containing all supported constraints fields of
-CertificateRequestPolicy.
-
-```yaml
-apiVersion: policy.cert-manager.io/v1alpha1
-kind: CertificateRequestPolicy
-metadata:
-  name: my-policy
-spec:
-  ...
-  constraints:
-    minDuration: 1h
-    maxDuration: 24h
-    privateKey:
-      algorithm: RSA
-      minSize: 2048
-      maxSize: 4096
-  ...
-```
-
-### Selector
-
-Selector is a required field that is used for matching
-CertificateRequestPolicies against a CertificateRequest for evaluation.
-approver-policy currently only supports selecting over the `issuerRef` of a
-request.
-
-`issuerRef` values accept wildcards "\*". If an `issuerRef` is set to an empty
-object "{}", then the policy will match against _all_ RBAC bound requests.
-
-```yaml
-apiVersion: policy.cert-manager.io/v1alpha1
-kind: CertificateRequestPolicy
-metadata:
-  name: my-policy
-spec:
-  ...
-  selector:
-    issuerRef:
-    - name: "my-ca"
-      kind: "*Issuer"
-      group: "cert-manager.io"
-```
-
-```yaml
-apiVersion: policy.cert-manager.io/v1alpha1
-kind: CertificateRequestPolicy
-metadata:
-  name: match-all-requests
-spec:
-  ...
-  selector:
-    issuerRef: {}
-```
-
-> ⚠️ Note that the user must still be bound by [RBAC](#Configuration) for
-> the policy to be considered for evaluation against a request.
-
-### Plugins
-
-Plugins are external approvers that are built into approver-policy at compile
-time. Plugins are designed to be used as extensions to the existing policy
-checks where the user requires special functionality that the existing checks
-can't provide.
-
-Plugins are defined as a block on the CertificateRequestPolicy Spec.
-
-```yaml
-apiVersion: policy.cert-manager.io/v1alpha1
-kind: CertificateRequestPolicy
-metadata:
-  name: plugins
-spec:
-  ...
-  plugins:
-    my-plugin:
-      values:
-        val-1: key-1
-```
+cert-manager [CertificateRequests](../concepts/certificaterequest/) can be
+rejected from being signed by using the [approval
+API](../concepts/certificaterequest/#approval).
+[approver-policy](https://github.com/cert-manager/approver-policy) is a
+cert-manager project that enables you to write policy to automatically manage
+this approval mechanism.
```

Comment:
I suggest not linking to the GitHub project here. We want to direct users to the cert-manager.io/projects page if that is to be the new home of the documentation.
```suggestion
approver-policy is a
cert-manager project that enables you to write policy to automatically manage
this approval mechanism.
```